internal/sirius/client.go

```diff
@@ -65,12 +87,12 @@ func (c *Client) get(ctx Context, path string, v interface{}) error {
 	if err != nil {
 		return err
 	}
-	defer resp.Body.Close() //#nosec G307 false positive
 
 	if resp.StatusCode != http.StatusOK {
 		return newStatusError(resp)
 	}
 
+	defer resp.Body.Close() //#nosec G307 false positive
```

Comment:
Was this supposed to be moved? If there's a non-200 error code then I think this now means the body won't get closed

Reply:
No, that was my mistake, I'll move it back. I probably need the same in TasksForCase().